mantle/cmd/kola/kola.go

```diff
@@ -648,7 +649,17 @@ func getParentFcosBuildBase(stream string) (string, error) {
 			return getParentFcosBuildBase("testing-devel")
 		}
 
-		parentVersion = index.Releases[n-1].Version
+		findParentVersion:
+  		for i := range index.Releases {
+				//to iterate through the list in reverse order
+    		release := index.Releases[(n - 1 - i)]
+      	for _, commit := range release.Commits {
+    			if commit.Architecture == (system.RpmArch()) {
```

Comment:
```suggestion
    			if commit.Architecture == (kola.Options.CosaBuildArch) {
```

in the return statement below we use `kola.Options.CosaBuildArch` so we should probably use that here too.

Reply:
addressed in recent commit. could you please have a look?